bears/vcs/git/GitCommitBear.py

```diff
@@ -34,14 +34,22 @@ class GitCommitBear(GlobalBear):
             'issue': r'(?:\w+/\w+)?#(\S+)',
             'full issue': r'https?://gitlab\S+/issues/(\S+)',
         },
+        'bitbucket': {
+            'issue': r'#(\S+)',
+            'full issue': None,
+        },
     }
     SUPPORTED_HOST_KEYWORD_REGEX = {
-        'github': (r'[Cc]lose[sd]?'
+        'github': (r'(?:[Cc]lose[sd]?'
```

Comment:
I do not think changes made to the regex of hosts other than BitBucket are either necessary or in the scope of this issue.

Reply:
Oh
Okay
Did that to support ```fixes bug#166``` but not ```fixesbug #11```
Will revert :sweat_smile: 

Thanks :D